recipes/nng/all/conanfile.py

```diff
@@ -20,13 +20,15 @@ class NngConan(ConanFile):
         "nngcat": [True, False],
         "http": [True, False],
         "tls": [True, False],
+        "maxTaskqThreads": "ANY"
```

Comment:
```suggestion
        "max_taskq_threads": "ANY"
```
Avoid camelCase in CCI.